components/mpas-seaice/testing_and_setup/couplercheck/check_seaice_budget_terms.py

```diff
@@ -0,0 +1,310 @@
+import subprocess
+import re
+import pandas as pd
+import xarray as xr
+import matplotlib.pyplot as plt
+from netCDF4 import date2num, Dataset
+import numpy as np
+from datetime import datetime
+import gzip, io, os, glob
+import argparse, sys
+
+verbose = False
+savedata = False
+
+def main():
```

Comment:
```suggestion
def main():
     global verbose, savedata
```
I'm not sure about the exact syntax but it seems likely to be right.  You only need to give `global` for module variables you will modify.  Otherwise, you can use them but you can't change them.